### ISSUE TITLE
Isolated execution of performance tests

### ISSUE THREAD

**Issue by nachtmaar** [kyma-project/test-infra#1025]
**Description**

The [performance test runner](https://github.com/venturasr/test-infra/blob/c5aa01de0ffcd60b1048c71fff699f7637a5313f/performance-tools/performance-cluster/runner.sh) provides a setup step for each component which shall be tested. However, it does not offer a cleanup step.

See also: https://github.com/kyma-project/test-infra/pull/899

**Reasons**

Each performance test deploys resources into the Kyma cluster. When executing the performance tests they should always yield the same result if no code changed the performance. It shouldn't be influenced by k8s resources consuming CPU or RAM.
In other words: each component should be tested in an isolation fashion.
For that isolation we need to cleanup all resources that the `setup.sh` created.
This ensures that all component tests are executed on a cluster with the same resource utilization.

**Attachments**

**Comment by derberg** [comment]
@nachtmaar Hey, any reason why you did not unstale it? is it obsolete and can be closed?
